### PR TITLE
Component tests

### PR DESCRIPTION
**Pull request by rosscreighton**
might want to go through this one commit by commit as i did some linting/package cleanup as i was working.

### CODE REVIEW

File: .meteor/dev_bundle [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/rosscreighton/.meteor/packages/meteor-tool/.1.3.4_4.13ws1vt++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle
```
Comment (sameckmeier) [inline .meteor/dev_bundle]: ya, im not sure either. that could be due to updating meteor

File: imports/ui/components/TestedComponent/TestedComponent.tests.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import { render } from 'enzyme';
+import TestedComponent from './TestedComponent';
+
+describe('<TestedComponent />', function() {
+  it('renders the testProp', function() {
+    const testPropValue = '1278hdq9d9283ydhq';
+    const wrapper = render(<TestedComponent testProp={testPropValue} />);
+    chai.expect(wrapper.text()).to.equal(testPropValue);
+  });
+});
```
Comment (sameckmeier) [inline imports/ui/components/TestedComponent/TestedComponent.tests.js]: 👍 not sure if chai makes it easy to navigate/query a html node tree. if not, it'd probably make our lives easier to find a package that does.

File: imports/ui/components/TestedComponent/TestedComponent.tests.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import TestedComponent from './TestedComponent';
 describe('<TestedComponent />', function() {
   it('renders the testProp', function() {
     const testPropValue = '1278hdq9d9283ydhq';
-    const wrapper = render(<TestedComponent testProp={testPropValue} />);
-    chai.expect(wrapper.text()).to.equal(testPropValue);
+    const rendered = render(<TestedComponent testProp={testPropValue} />);
+    chai.expect(rendered.text()).to.equal(testPropValue);
   });
 });
```
Comment (sameckmeier) [inline imports/ui/components/TestedComponent/TestedComponent.tests.js]: nice change. we could also call it html. we might want to think about building test utils that make it quicker/encapsulate that code we use to stub our components.

File: .meteor/dev_bundle (hunk 1 above)
Comment (rosscreighton) [inline .meteor/dev_bundle]: when i ran meteor update later on, this file was added to .gitnore, so i think we're good

File: imports/ui/components/TestedComponent/TestedComponent.tests.js (hunk 2 above)
Comment (rosscreighton) [inline imports/ui/components/TestedComponent/TestedComponent.tests.js]: ah, yeah, `enzyme` byb airbnb is the package i'm using for this. from the enzyme docs: 

"Enzyme is a JavaScript Testing utility for React that makes it easier to assert, manipulate, and traverse your React Components' output.

Enzyme's API is meant to be intuitive and flexible by mimicking jQuery's API for DOM manipulation and traversal."

File: imports/ui/components/TestedComponent/TestedComponent.tests.js (hunk 3 above)
Comment (rosscreighton) [inline imports/ui/components/TestedComponent/TestedComponent.tests.js]: thats a good idea. we should definitely keep that in mind as we write more tests. i would say we should start thinking about it now, but i really don't know what those utilities would look like as i've only written one test so far. i written JS tests at TSF either

**Comment by rosscreighton** [comment]
@sameckmeier is this ok to merge?

**Comment by sameckmeier** [comment]
@rosscreighton looks good to merge

**Comment by rosscreighton** [comment]
@sameckmeier cool, i'm going to merge the example component for now so we have a reference, but we should remove it soon

**Pull request merged**